packages/performance-tests/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "transformer-performance-tests",
+  "private": true,
+  "license": "MIT",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "tsc 1>&2",
+    "build:ci": "npm run -s build",
+    "clean": "rimraf lib",
+    "lint": "eslint --max-warnings 0 \"./src/**/*.ts\" 1>&2",
+    "test": "mocha --delay --timeout 300000 --require ts-node/register test/**/*.test.ts",
+    "test-reg": "mocha --delay --timeout 300000 --require ts-node/register test/**/TransformerRegression.test.ts",
+    "test-mocha": "mocha --delay --timeout 300000 \"./lib/**/TransformerRegression.test.js\"",
+    "process-reports": "node scripts/process-reports"
+  },
+  "repository": {},
+  "dependencies": {
+    "@itwin/core-backend": "3.6.0 - 3.6.1",
```

Comment:
these dependencies are not aligned with those in `main`. Please update them to the ones in main, which should be something like `3.6.0 - 3.7.999`, don't forget dev deps

Reply:
have to update them to 4.0.999 now :) main had another PR go through :)

Reply:
done